test/hotspot/jtreg/serviceability/jvmti/events/MonitorContendedEnter/mcontenter01/libmcontenter01.cpp

```diff
@@ -71,7 +70,7 @@ MonitorContendedEnter(jvmtiEnv *jvmti, JNIEnv *jni, jthread thr, jobject obj) {
 
 /* ========================================================================== */
 
-static int prepare() {
+static int prepare(JNIEnv* jni) {
```

Comment:
Is this a bug fix?

Reply:
Testing ran into a couple of bugs in JVMTI tests. One of was tests that was stashing the JNIEnv into a static.